modules/editable-layers/package.json

```diff
@@ -64,10 +64,14 @@
     "@turf/union": "^6.5.0",
     "cubic-hermite-spline": "^1.0.1",
     "eventemitter3": "^5.0.0",
-    "geojson-types": "^2.0.1",
+    "geojson": "^0.5.0",
     "lodash.throttle": "^4.1.1",
     "uuid": "9.0.0",
-    "viewport-mercator-project": ">=6.2.3"
+    "viewport-mercator-project": ">=6.2.3",
+    "boxicons": "^2.1.4"
```

Comment:
I don't see where these packages are used. Can you explain why you added them ?

Reply:
It has been used in the file: modules/react/src/components/icon.tsx

Reply:
I have moved it into the package.json of modules/react

Reply:
I don't understand, you did not modify the file icon.tsx, how did the project compiled before you add the package boxicons ?

Reply:
Because, they only build the modules/editable-layers package

Reply:
I don't think it's a good idea to add this package when the modifications that were done are not related to it. 
If you have a good reason to add it, it needs to be done in a separate PR.

Reply:
If don't have it, then we have an error